index.html

```diff
@@ -228,11 +247,11 @@ <h2 class="headline__title headline__title--half">About Myself</h2>
 
     <section class="contact">
       <div class="contact-wrapper">
-        <img
+        <!-- <img
           class="contact-shapes"
           src="images/contact-form-shapes-mobile.svg"
           alt="Contact form shapes"
-        />
+        /> -->
```

Comment:
## [OPTIONAL]
- [ ] Kindly remove all commented code. this will make your code clean and more readable.